src/Components/Share/ShareModal.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState, useEffect } from "react";
+
+const ShareModal = ({ url, id }) => {
+  const urlInputId = `${id}-share-url`;
```

Comment:
We give the input a unique ID (so if we render the component twice they won't clash)